docs/shipping/Code/node-js.md

```diff
@@ -56,18 +56,68 @@ var logger = require('logzio-nodejs').createLogger({
-| protocol | `http` or `https`. The value of this parameter affects the default of the `port` parameter. | `http` |
+| protocol | `http`, `https` or `udp`. The value of this parameter affects the default of the `port` parameter. | `http` |
 | host  |  {@include: ../../_include/log-shipping/listener-var.md} Replace `<<LISTENER-HOST>>` with the host [for your region](https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/#available-regions). The required port depends whether HTTP or HTTPS is used: HTTP = 8070, HTTPS = 8071. | `listener.logz.io` |
-| port | Destination port. The default port depends on the `protocol` parameter: `8070` (for HTTP) or `8071` (for HTTPS) | `8070` / `8071` |
+| port | Destination port. The default port depends on the `protocol` parameter. For udp default port is `5050`, http is `8070`, and https is `8071` | `5050` / `8070` / `8071` |
 | type | {@include: ../../_include/log-shipping/type.md} | `nodejs` |
 | sendIntervalMs  | Time to wait between retry attempts, in milliseconds. | `2000` (2 seconds) |
 | bufferSize  | Maximum number of messages the logger accumulates before sending them all as a bulk. | `100` |
 | numberOfRetries | Maximum number of retry attempts. | `3` |
 | debug | Set to `true` to print debug messages to the console.  | `false` |
-| callback | A callback function to call when the logger encounters an unrecoverable error. The function API is `function(err)`, where `err` is the Error object. | -- |
+| callback | A callback function to call when the logger encounters an unrecoverable error. On success: `callback()`. On error: `callback(error)` where `error` is the Error object. This function enables you to handle errors and successful transmissions independently. | -- |
 | timeout | Read/write/connection timeout, in milliseconds. | -- |
 | extraFields | JSON format. Adds your custom fields to each log. Format: `extraFields : { field_1: "val_1", field_2: "val_2" , ... }` | -- |
 | setUserAgent | Set to false to send logs without the user-agent field in the request header.  | `true` |
+| addOtelContext | Add `trace_id`, `span_id`, and `service_name` fields to logs when OpenTelemetry context is available. | Default: `true` |
+
+### Using UDP
+
+While the UDP protocol can be used, it has limitations that make it less than ideal:
+
+* Lack of Reliability: UDP does not guarantee that logs are received, making it less reliable than other protocols.
+* Suboptimal Performance: UDP cannot utilize the bulk API, leading to reduced performance.
+
+When using UDP, each message is sent individually rather than in batches. However, the `bufferSize` parameter still applies differently:
+
+The logger waits for the buffer to reach the specified size before sending all messages. To send each message immediately, set `bufferSize = 1`.
+
+For better reliability and performance, consider using a different protocol when possible.
+
+### Callback usage
```

Comment:
Not sure if we need this on our official docs site, the docs are already very long @Simplychee its your call :)